src/ui/components/Header/RecordingTrialEnd.tsx

```diff
@@ -8,7 +8,15 @@ export function RecordingTrialEnd() {
   const { recording, loading } = hooks.useGetRecording(recordingId);
 
   const workspace = recording?.workspace;
-  if (loading || !workspace || !inUnpaidFreeTrial(workspace) || workspace.hasPaymentMethod) {
+  const metadata = recording?.metadata;
+
+  if (
+    loading ||
+    !workspace ||
+    metadata?.test ||
+    !inUnpaidFreeTrial(workspace) ||
+    workspace.hasPaymentMethod
+  ) {
```

Comment:
Let's change this to check the workspace instead of the recording. So don't show the trial if it's a test workspace, i.e. `workspace.isTest === true`.